code/modules/clothing/head/f13factionhead.dm

```diff
@@ -251,14 +251,14 @@
 
 /obj/item/clothing/head/helmet/f13/legion/prime
 	name = "legion prime helmet"
-	desc = "(IV) A helmet from reinforced leather witha red peak."
+	desc = "(IV) A helmet from reinforced leather with a red peak."
 	item_state = "legion-prime"
 	icon_state = "legion-prime"
 	armor = list("tier" = 4, "energy" = 15, "bomb" = 25, "bio" = 40, "rad" = 20, "fire" = 60, "acid" = 0)
 
 /obj/item/clothing/head/helmet/f13/legion/prime/slavemaster
-	name = "slave master helmet"
-	desc = "(IV) The unique helmet used by the disciplinary corps of the Legion."
+	name = "slavemaster helmet"
+	desc = "(IV) gladiatorial style helmet."
```

Comment:
Why.

Reply:
Because Slavemaster is either a part of a special "elite" disciplinary corps or a off duty temporary assignment, it can't really be both. The description is pretty bare bones I admit but feel free to make a suggestion, its a very, very  minor thing.